Tools/OSD/osd_screen.cs

```diff
@@ -604,12 +606,6 @@ private void rbtSortAlphabetic_CheckedChanged(object sender, EventArgs e)
 			}
 		}
 		
-        private int correct_NTSC(int y){
-            if (y >= osd.getCenter() && osd.is_ntsc()) 
-                y+=3;
-            return y;
-        }
-
```

Comment:
This function adds 3 lines to the object every time you click on a node item.

I'm going to refactor the whole class to be OOP, don't worry about this